spec/site_prism/page_objects/page/results.rb

```diff
@@ -22,12 +22,19 @@ class VacanciesSection < SitePrism::Section
       element :link, '[data-qa="link"]'
     end
 
+    class FundingSection < SitePrism::Section
+      element :subheading, "h2"
```

Comment:
:eyes: this should have `data-qa` on it

Reply:
I removed this (and some other unused bits) since we are not using it for testing. Hope that addresses your comment